src/components/Pokemon.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+class Pokemon extends React.Component {
```

Comment:
There's not really a need to use class components. Depending on your comfort level we could move to hooks. We can have a mix of both too while you get used to them. https://reactjs.org/docs/hooks-intro.html

Reply:
I have no experience with Hooks! I typically use class components for components that manage state (i.e. containers) and functional components for components that are just presentational.